static/src/javascripts/projects/commercial/modules/cmp/cmp.js

```diff
@@ -279,7 +279,7 @@ class CmpService {
 
 export const init = (): void => {
     // Only run our CmpService if prepareCmp has added the CMP stub
-    if (window[CMP_GLOBAL_NAME] && !isInCcpaTest()) {
+    if (window[CMP_GLOBAL_NAME] && !isCcpaApplicable()) {
```

Comment:
@ripecosta i'm not sure what `window[CMP_GLOBAL_NAME]` does, do we still want it here?

Reply:
This line and `window[CMP_GLOBAL_NAME]` is checking if `windows.__cmp` exists (meaning that the stub was added to the page) before loading the framework fully. I'd advise keeping it because when we roll out TCFv2 to every user we will delete this entire file. So no use spending time tweaking the logic now and risk unintended consequences when we're gonna get rid of the whole thing soon.